docs/readthedocs/index.md

```diff
@@ -1,20 +1,24 @@
-Welcome to simona docs
-======================
+# Welcome to simona docs
 
+```{eval-rst}
 .. figure:: ../logo/logo_tightcrop_transparent.png
   :figwidth: 25%
   :align: right
   :alt: logo of simona
+```
+
```

Comment:
Guess one empty line can be removed here, which would solve one of the codacy issues.